Rakefile

```diff
@@ -47,4 +47,22 @@ namespace 'smithy-client' do
     t.ruby_opts = '-I gems/smithy-client/spec'
     t.rspec_opts = '--format documentation'
   end
+
+  task 'rbs:validate' do
+    sh('bundle exec rbs -I gems/smithy-client/sig validate')
+  end
+
+  task 'rbs:test' do
+    env = {
+      'RUBYOPT' => '-r bundler/setup -r rbs/test/setup',
+      'RBS_TEST_RAISE' => 'true',
+
+      'RBS_TEST_OPT' => '-I gems/smithy-client/sig',
+      'RBS_TEST_TARGET' => '"Smithy,Smithy::*,Smithy::Client,Smithy::Client::*"'
```

Comment:
Does Smithy::* not already capture those?

Reply:
I've been playing around with this - I tried every possible combo.  I was able to remove `Smithy::Client::*` and still get the correct errors.  But if I remove any others then it does not work.  I don't understand why, but maybe you can try as well.

Reply:
I'm sure it's fine. Weird that it doesn't include children modules.